core/arch/arm/plat-nuvoton/conf.mk

```diff
@@ -12,6 +12,8 @@ $(call force,CFG_GIC,y)
 $(call force,CFG_ARM_GICV2,y)
 $(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
 $(call force,CFG_16550_UART,y)
+$(call force,CFG_EXTERNAL_DT,n)
+$(call force,CFG_WARN_INSECURE,n)
```

Comment:
It would be nice to have this change in a separate commit with an explanation of why it's not needed.